pkg/operator/ceph/cluster/controller.go

```diff
@@ -486,11 +486,11 @@ func (c *ClusterController) onK8sNodeUpdate(oldObj, newObj interface{}) {
 
 	// Checking for NoSchedule added to storage node
 	if oldNodeSchedulable == false && newNodeSchedulable == false {
-		logger.Debugf("Skipping cluster update. Updated node %s was and is still unschedulable", newNode.Labels[v1.LabelHostname])
+		//logger.Debugf("Skipping cluster update. Updated node %s was and is still unschedulable", newNode.Labels[v1.LabelHostname])
```

Comment:
Why comment them out instead of removing them?

Reply:
I was debating too... Maybe i'll convert to normal comments.